doc/iterations/2024.3/shapes/overflow_label_strategy.adoc

```diff
@@ -0,0 +1,61 @@
+= Overflow label strategy
+
+== Problem
+
+In the existing node layouts, the inside label of a node will always be used to determine the minimum size.
+This means that in the case of a long label, the node will necessarily take a size that allows the label to be displayed without any modification.
+In some cases, specifier doesn't want that the label affects the node representation.
+
+== Key Result
+
+Allow the specifier to define a label that applies an auto wrap rather than enlarging the node.
+A label should be broken at any point if there are no otherwise-acceptable break points (like a long word or URL).
+
+== Solution
+
+Add to the view DSL, for inside label:
+
+* _maxLabelWidthExpression_
+* _overflowStrategy_: none, wrap or ellipsis
```

Comment:
Why not wrap+ellipsis?

- none: The node is horizontally resized (but not vertically) to display its label
- wrap: The node is not resized horizontally, the label is wrapped at the end of the line. The node can be resized vertically by its label.
- ellipsis: The node is not resized horizontally or vertically, the label is ending with an ellipsis at the end of the line
- wrap+ellipsis: The label is wrapped at the end of the line but it does not change the size of the node vertically either. An ellipsis is displayed if the node is full.